keyboards/keebio/quefrency/rev6/config.h

```diff
@@ -0,0 +1,27 @@
+// Copyright 2024 Danny Nguyen (@nooges)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+/* Defines for the split keyboard setup */
+#define SERIAL_USART_DRIVER SD3         // USART 3
+#define SERIAL_USART_TX_PIN B10
+#define SERIAL_USART_RX_PIN B11
+#define SERIAL_USART_TX_PAL_MODE 7
+#define SERIAL_USART_RX_PAL_MODE 7
+#define SERIAL_USART_FULL_DUPLEX
+#define SERIAL_USART_PIN_SWAP
+
+#define USB_VBUS_PIN C6
+#define SPLIT_HAND_PIN A0
+
+/* Defines for the RGB matrix */
+#ifdef RGB_MATRIX_ENABLE
+#   define NOP_FUDGE 0.4
```

Comment:
This shouldn't be needed? Compiles fine without and it appears you have WS2812 driver and the like configured appropriately.

Reply:
Removed it now after testing w/o it